cli/lsp/tsc.rs

```diff
@@ -2326,6 +2327,7 @@ fn parse_code_actions(
             update_import_statement(
               tc.as_text_edit(asset_or_doc.line_index()),
               data,
+              Some(&language_server.get_ts_response_import_mapper()),
```

Comment:
I resisted the urge to do any drastic refactoring here. I'd like to improve this code in the future.